public/index.css

```diff
@@ -0,0 +1,14 @@
+#modalbox {
+  display: none;
+  position: fixed;
+  z-index: 400;
+  left: 50%;
+  top: 50%;
+  width: 20%; /* Full width */
```

Comment:
They're no longer full width and full height, so update the comments to reflect that.